modules/ROOT/pages/asm-secret-group-concept.adoc

```diff
@@ -3,9 +3,8 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A secret group is a unique secrets vault to which all your secrets are bound. Each secret group has individual access and authorization controls.
-
-Secrets Manager stores your secrets per secret group, per environment and per business group. User access and authorization controls are applied at the environment level. A user who has access on a given environment can access all the secret groups that belong to that environment. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
+A secret group is a unique secrets vault to which all your secrets are bound. Secrets Manager stores your secrets per secret group, per environment, and per business group.
+User access and authorization controls are applied at the environment level. A user who has access on a given environment can access all the secret groups that belong to that environment. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
```

Comment:
If you want the **User access and authorization (...)** part to be in a different paragraph , you need to move that to a new line preceded by a blank line:
```
A secret group is a unique secrets vault to which all your secrets are bound. Secrets Manager stores your secrets per secret group, per environment, and per business group.

User access and authorization controls are applied at the environment level. (...)
```
Otherwise, it will build as a line after the other:
```
secret group, per environment, and per business group. User access and authorization controls are applied at the environment level. (...)
```